terraform-post-packer/variables.tf

```diff
@@ -11,7 +11,7 @@ variable "extraorg_account_ids" {
 }
 
 variable "recent_ami_count" {
-  default     = 12
+  default     = 24
```

Comment:
We should probably update the description to reflect that this is really the last 12 for each architecture. Alternatively we could bre\ak up the `historical_amis` resource into architecture-specific resources (but this is probably more work).

Reply:
Please see commit [89421b1](https://github.com/cisagov/skeleton-packer/pull/355/commits/89421b11db1d8b0081ca6b5d5354d7d85f5fd9a4).

Reply:
Thanks for the updated description. The more I think about it the more I think we should take the approach of separating them however. I believe since we're using a `toset()` and `slice()` we could just do separate `data` resources for each and merge them in the `toset()` for the module.

Reply:
I reworked this PR.  Please see the commits below.

Reply:
@mcdonnnj - Can you take another look at this PR?  I believe I have addressed your concerns.